projects/js-toolkit/packages/js-toolkit-core/src/project/liferayCli/Build.ts

```diff
@@ -8,15 +8,17 @@ import fs from 'fs';
 import FilePath from '../../file/FilePath';
 import LiferayJson, {
 	BuildConfig,
+	BuildType,
```

Comment:
Mmm, this was duplicated on purpose. The reason being that `liferay.json`'s schema is a contract between us and our users and the build type is internal to `@liferay/cli` and may change in the future.

So, for example, we could have three different types of builds in `liferay.json` that map to the same internal build in `@liferay/cli`.

However, that may be achieved at code level too maybe (by delegating). So I'm OK with deduplicating it for now, as long as we agree on not changing `LiferayJson` in the future because that would break customer projects.

Note that it is also possible that we end up abandoning liferay.json if we finally converge with blade, so... :shrug: